python/hail/dataset.py

```diff
@@ -2819,7 +2819,8 @@ def lmmreg(self, kinshipMatrix, y, covariates=[], global_root="global.lmmreg", v
         | ``va.lmmreg.pval``     | Double | :math:`p`-value                                                         |
         +------------------------+--------+-------------------------------------------------------------------------+
 
-        Those variants that don't vary across the included samples (e.g., all genotypes are HomRef) will have missing annotations.
+        :py:meth:`.lmmreg` skips variants that don't vary across the included samples,
```

Comment:
The previous comment seemed more clear.  What does it mean to skip a variant?  You use the word missing when discussing filters.